src/js/models/image.js

```diff
@@ -12,11 +11,7 @@ export default class Image extends Section {
     return false;
   }
 
-  headPosition() {
-    return new Position(this, 0);
-  }
-
-  tailPosition() {
-    return new Position(this, 1);
+  get length() {
```

Comment:
IMO this whole model should go the way of the Dodo before 1.0. images are just cards.